src/NUnitFramework/tests/Constraints/MsgUtilTests.cs

```diff
@@ -124,23 +124,19 @@ public static void FormatValue_CharTest(char c, string expected)
             Assert.That(MsgUtils.FormatValue(c), Is.EqualTo(expected));
         }
 
-        [TestCase(null, null, "[null, null]")]
-        [TestCase(null, "Second", "[null, \"Second\"]")]
```

Comment:
I guess these are removed (and the same comment for the test below) as the key must not be `null`?

Reply:
Yes, indeed. `Dictionary<TKey, TValue> where TKey : notnull`.